kv/dist_sender.go

```diff
@@ -578,7 +592,11 @@ func (ds *DistSender) sendAttempt(trace *tracer.Trace, args proto.Request, desc
 
 	// If this request needs to go to a leader and we know who that is, move
 	// it to the front.
-	if !(proto.IsRead(args) && args.Header().ReadConsistency == proto.INCONSISTENT) &&
+	// TODO(tschottdorf): replace IsRead() by IsReadOnlyBatch or something like that,
```

Comment:
something wonky about this comment since `IsReadOnlyBatch` is already used below